public/app/features/dashboard-scene/scene/DashboardReloadBehavior.ts

```diff
@@ -56,28 +76,28 @@ export class DashboardReloadBehavior extends SceneObjectBase<DashboardReloadBeha
   }
 
   private reloadDashboard() {
-    if (!this.isEditing() && !this.isWaitingForVariables()) {
-      const timeRange = sceneGraph.getTimeRange(this);
-
-      // This is wrapped in setTimeout in order to allow variables and scopes to be set in the URL before actually reloading the dashboard
-      setTimeout(() => {
-        getDashboardScenePageStateManager().reloadDashboard({
-          version: this.state.version!,
-          scopes: getClosestScopesFacade(this)?.value.map((scope) => scope.metadata.name) ?? [],
-          // We're not using the getUrlState from timeRange since it makes more sense to pass the absolute timestamps as opposed to relative time
-          timeRange: {
-            from: timeRange.state.value.from.toISOString(),
-            to: timeRange.state.value.to.toISOString(),
-          },
-          variables: sceneGraph.getVariables(this).state.variables.reduce<UrlQueryMap>(
-            (acc, variable) => ({
-              ...acc,
-              ...variable.urlSync?.getUrlState(),
-            }),
-            {}
-          ),
-        });
-      });
+    if (this.isEditing() || this.isWaitingForVariables()) {
+      return;
     }
+
+    // This is wrapped in setTimeout in order to allow variables and scopes to be set in the URL before actually reloading the dashboard
+    setTimeout(() => {
+      getDashboardScenePageStateManager().reloadDashboard({
+        version: this.state.version!,
+        scopes: this._scopesBridge?.getValue().map((scope) => scope.metadata.name) ?? [],
+        // We're not using the getUrlState from timeRange since it makes more sense to pass the absolute timestamps as opposed to relative time
+        timeRange: {
+          from: this._timeRange!.state.value.from.toISOString(),
+          to: this._timeRange!.state.value.to.toISOString(),
+        },
+        variables: sceneGraph.getVariables(this).state.variables.reduce<UrlQueryMap>(
+          (acc, variable) => ({
+            ...acc,
+            ...variable.urlSync?.getUrlState(),
+          }),
+          {}
```

Comment:
you should be able to use  `sceneUtils.getUrlState(dashboard)`. to get the full url state of the dashboard